assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -248,18 +254,19 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
       pinchable={true}
       keepRatio={isResizingFromCorner}
       renderDirections={getRenderDirections(resizeRules)}
-      snappable={true}
-      snapElement={true}
-      snapHorizontal={true}
-      snapVertical={true}
-      snapCenter={true}
+      snappable={canSnap}
+      snapElement={canSnap}
+      snapHorizontal={canSnap}
+      snapVertical={canSnap}
+      snapCenter={canSnap}
       horizontalGuidelines={
-        actionsEnabled ? [0, canvasHeight / 2, canvasHeight] : []
+        canSnap && actionsEnabled ? [0, canvasHeight / 2, canvasHeight] : []
       }
       verticalGuidelines={
-        actionsEnabled ? [0, canvasWidth / 2, canvasWidth] : []
+        canSnap && actionsEnabled ? [0, canvasWidth / 2, canvasWidth] : []
       }
-      elementGuidelines={actionsEnabled ? otherNodes : []}
+      elementGuidelines={canSnap && actionsEnabled ? otherNodes : []}
+      isDisplaySnapDigit={false}
```

Comment:
Hmm that doesn't seem to work, I can still see numbers when snapping to another element

Reply:
@swissspidy did you re-rerun `npm install`? The change is working here.

![Kapture 2020-03-05 at 11 19 07](https://user-images.githubusercontent.com/1680157/75990127-2f20a380-5ed3-11ea-8fc8-20c2b4948ec0.gif)

Reply:
Thanks, that worked 🤦‍♂ 

Didn't realize this was just added in the last version.